verilog/analysis/verilog_linter.cc

```diff
@@ -55,6 +55,8 @@
 ABSL_FLAG(verilog::RuleBundle, rules, {},
           "List of lint rules to enable.  "
           "Prefix a rule name with '-' to disable it.");
+ABSL_FLAG(std::string, rules_config, ".rules.verible_lint",
+          "Path to lint rules configuration file.");
```

Comment:
Let's make it clear whether this is an absolute or relative path.

Reply:
It's actually both right now. This help will need elaboration when we implement the search feature.

Reply:
Keep in mind when the config-search is implemented, we will want a way to disable all searching behavior.  (e.g. an empty `rules_config` string might suffice.)

Reply:
Sure, I will keep that in mind in the following PRs, thanks!